x-pack/test/api_integration/apis/uptime/rest/get_monitor_overview.ts

```diff
@@ -21,26 +22,26 @@ export default function ({ getService }: FtrProviderContext) {
     let _monitors: MonitorFields[];
     let monitors: MonitorFields[];
 
-    const deleteMonitor = async (id: string) => {
-      try {
-        await supertest
-          .delete(`${API_URLS.SYNTHETICS_MONITORS}/${id}`)
-          .set('kbn-xsrf', 'true')
-          .expect(200);
-      } catch (e) {
-        // eslint-disable-next-line no-console
-        console.error(e);
-      }
-    };
-
-    const saveMonitor = async (monitor: MonitorFields) => {
-      const res = await supertest
-        .post(API_URLS.SYNTHETICS_MONITORS)
-        .set('kbn-xsrf', 'true')
-        .send(monitor);
-
-      return res.body as SimpleSavedObject<MonitorFields>;
-    };
+    // const deleteMonitor = async (id: string) => {
```

Comment:
I think you meant to delete this.

Reply:
It is weird that this is there, I'm wondering if I messed up merging something.

Anyways, sorry about that.